src/desktop/analytics/artwork.js

```diff
@@ -522,9 +522,4 @@ import { data as sd } from "sharify"
     .on("artwork:contact-specialist", function() {
       analytics.track("Clicked 'Contact Artsy Specialist'")
     })
-    .on("artwork:confirm-registration", function() {
```

Comment:
Do we know if there's any corresponding instrumentation in Reaction now? Was the only call-sight to this hook from code that was deleted in this PR? Just wanna make sure we're not deleting an analytics event

Reply:
Yes, we have page tracking in reaction. I don't see any explicit examples of this event triggered in either Force or Reaction. Might have been fired dynamically using the route and model passed in. Not sure but either way should be safe to remove.

Reply:
Yeah, I guess it wasn't be used for a while anyways in Force, even if the code was there.